src/Zope2/utilities/mkwsgiinstance.py

```diff
@@ -218,7 +218,7 @@ def get_zope2path(python):
         output = subprocess.check_output(
             [python, '-c', 'import Zope2; print(Zope2.__file__)'],
             stderr=subprocess.PIPE)
-        zope2file = output.strip()
+        zope2file = output.strip().decode('utf-8')
```

Comment:
I think it should be decoded for Python 3 only; the return value from this function ends up getting used in a str.replace on the content of files opened in text mode.

Reply:
`str.decode` returns `unicode` on Python2.x, which is what we want, I believe.

Reply:
I don't think so. The line that broke in Python 3 in #372 was:

    text = text.replace("<<%s>>" % k, replacements[k])

where `replacements[k]` comes from the `zope2file` value.

`text` is a `str` so we want `zope2file` to also be a `str`, which means not decoding it in Python 2.